cargo-miri/src/phases.rs

```diff
@@ -222,8 +222,8 @@ pub fn phase_rustc(mut args: impl Iterator<Item = String>, phase: RustcPhase) {
     /// never set for host crates. This matches what rustc bootstrap does,
     /// which hopefully makes it "reliable enough". This relies on us always
     /// invoking cargo itself with `--target`, which `in_cargo_miri` ensures.
-    fn is_target_crate() -> bool {
-        get_arg_flag_value("--target").is_some()
+    fn non_host_target_crate() -> Option<String> {
```

Comment:
"target" and "host" are mutually exclusive, so I don't understand the point of the rename here.

Reply:
Every build has a "target" in the sense that it targets some target. I wanted to make it clear that this is only available when `--target` is passed but I agree that the name isn't good.

Reply:
I removed the target check anyways so this doesn't need to be touched anymore, `is_target_crate` is a fine name when it returns a `bool`.

Reply:
Note that `is_target_crate` is sometimes `true` even when target==host -- cargo has a distinction of host and target builds (these are my terms, not sure what they call it) that is different from whether target==host. It correlates with whether `--target` is passed to rustc.